WebClock/Controllers/ClockInOutController.cs

```diff
@@ -9,10 +11,12 @@ namespace WebClock.Controllers
     public class ClockInOutController : ControllerBase
     {
         private readonly IRepository _repository;
+        private readonly ClockInOutRepository _clockInOutRepository;
 
         public ClockInOutController(IRepository repository)
         {
             _repository = repository;
+            _clockInOutRepository = new ClockInOutRepository(_repository);
```

Comment:
Jeśli to "repozytorium" jest używane tylko w metodzie Balance, to ja bym tam umieścił jego tworzenie.